src/module/AddRemove.js

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable no-loop-func */
```

Comment:
- [ ] Kindly avoid disabling linters because they guide us in writing code that is clean and well formatted.